### PR TITLE
feat(route): dynamic creator route

### PR DESCRIPTION
**Pull request by dsope05**

### CODE REVIEW

File: components/services.tsx [hunk 1]
```diff
@@ -72,16 +86,20 @@ const MerchView= () => (
...
   const { loading, error, data } = useQuery(USER_CREATOR, {
-    variables: { id: 3 }
+    skip: !handle,
+    variables: { handle }
   });
   const services = data?.userCreators?.items?.[0]?.services ?? [];
   const servicesDisplay = services.map((service) => {
-    const { reviewAverage, description, title, tiers: [bronzeTier] } = service;
+    const { reviewAverage, description, title, tiers: [bronzeTier], photos } = service;
+    console.log('service', service)
```
Comment (aelzeiny) [inline components/services.tsx:98]: We should probably not have this in our logs.

File: components/creatorCard/creatorInfo.tsx [hunk 2]
```diff
@@ -3,12 +3,16 @@ import Image from 'next/image'
 import CreatorButton from '../utils/button'
 import { useQuery } from '@apollo/client';
 import { USER_CREATOR } from '../../graphql/queries';
+import { useRouter } from 'next/router';
 
 export default function CreatorInfo() {
+  const { profile: handle } = useRouter().query;
+  console.log('profile handle', handle)
```
Comment (aelzeiny) [inline components/creatorCard/creatorInfo.tsx:10]: We should probably not have this in our logs.

File: components/creatorCard/index.tsx [hunk 3]
```diff
@@ -18,8 +19,8 @@ export default function CreatorCard() {
          />
         </div>
         <div className={styles.creatorBtns}>
-          <CreatorButton className={styles.businessBtn} type="primary" text="Business" />
-          <CreatorButton type="secondary" text="Donate" />
+          <CreatorButton onClick={() =>{}} className={styles.businessBtn} type="primary" text="Business" />
```
Comment (aelzeiny) [inline components/creatorCard/index.tsx:22]: Is there a reason for empty callback?

File: pages/[profile].tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import styles from '../styles/Profile.module.css'
+import Header from '../components/header'
+import CreatorCard from '../components/creatorCard'
+import Services from '../components/services'
+import { useQuery } from '@apollo/client';
+import { useRouter } from 'next/router';
+
+export default function Profile() {
+  const { profile } = useRouter().query;
```
Comment (aelzeiny) [inline pages/[profile].tsx:9]: noice

**Review by aelzeiny — APPROVED** [review]
Assuming you fix the little nits, you're good to merge when you feel ready.

File: components/creatorCard/index.tsx (hunk 3 above)
Comment (dsope05) [reply components/creatorCard/index.tsx:22]: next was giving error when it was empty

Comment (dsope05) [reply components/creatorCard/index.tsx:22]: next was giving error when it was empty

**Pull request merged**